build_projects/shared-build-targets-utils/BuildPackageVersions.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Cli.Build
+{
+    public class BuildPackageVersions
```

Comment:
How about "DependencyVersions"?  I'm not sure I understand what "BuildPackage"s are.